qa/rpc-tests/wallet_shieldcoinbase.py

```diff
@@ -214,12 +214,13 @@ def run_test (self):
         assert_equal(result["shieldingUTXOs"], Decimal('50'))
         assert_equal(result["remainingUTXOs"], Decimal('50'))
         self.wait_and_assert_operationid_status(0, result['opid'])
+        sync_mempools([self.nodes[0]])
```

Comment:
We introduce a change similar to the one you have here in  "Made lisstransactions filterable by address #210 " pull request.
Please bear in mind you may have a conflict to be solved on merge.

Reply:
:+1: 

I will rebase on top of development after #209 #210 #211 and #223 are merged.

A test build with all merges combined is running here https://travis-ci.com/ZencashOfficial/zen/builds/147921839 already.

Reply:
Rebase done, offending commit removed in favor of https://github.com/ZencashOfficial/zen/commit/809d0503ed117a2fac824928893a5164c55c2dc8